lib/alephant/broker/cache/cached_object.rb

```diff
@@ -31,14 +31,14 @@ def updated
           time = metadata[:'head_Last-Modified']
           Time.parse(time)
         rescue TypeError, ArgumentError => error
-          logger.info(event: 'ErrorCaught', method: "#{self.class}#updated", error: error)
+          logger.error(method: "#{self.class}#updated", error: error)
```

Comment:
Why did we take event out? I thought event was a short description of the error.

Reply:
I think we should just use the information in `error`